cmd/imageproxy/main.go

```diff
@@ -45,6 +45,7 @@ var addr = flag.String("addr", "localhost:8080", "TCP address to listen on")
 var whitelist = flag.String("whitelist", "", "comma separated list of allowed remote hosts")
 var referrers = flag.String("referrers", "", "comma separated list of allowed referring hosts")
 var baseURL = flag.String("baseURL", "", "default base URL for relative remote URLs")
+var imagesPath = flag.String("imagesPath", "", "Path for serving images from file system")
```

Comment:
instead of an "imagesPath", what if baseURL is using the file protocol like: file://some/path/to/file and you just get the images from this root dir?

Reply:
yeah, i thought about it but need to check how it could possibly affect performance, because transport is creating on initialisation (cmd/imageproxy/main.go:64) and I can't imagine in which cases it will be useful to handle both situations (fs and external images)

Reply:
ok :).